packages/playwright/src/worker/workerMain.ts

```diff
@@ -250,7 +250,9 @@ export class WorkerMain extends ProcessRunner {
         attachment => this.dispatchEvent('attach', attachment));
 
     const processAnnotation = (annotation: Annotation) => {
-      testInfo.annotations.push(annotation);
+      // testInfo.slow() does the same thing, hence we skip it here.
+      if (annotation.type !== 'slow')
```

Comment:
I'd rather call `testInfo._timeoutManager.slow()` instead of `testInfo.slow()` below.

Ideally, we wouldn't have duplication here, for example we can call `testInfo._modifier()` but make it not throw.